### ISSUE TITLE
Add a README

### ISSUE THREAD

**Issue by giggio** [ScottIsAFool/LiveWriterPluginManager#5]
It should explain how to install and use the plugin manager.

**Comment by maphew** [comment]
Yes please. I've downloaded v0.2.0.2 of Setup.msi and run it. Now what? I don't seem to have any new menu items in Open Live Writer or Windows Start.
